packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -151,10 +151,16 @@ export class MLSService extends TypedEventEmitter<Events> {
     client: RegisteredClient,
     {skipInitIdentity, ...mlsConfig}: InitClientOptions,
   ) {
+    // filter out undefined values from mlsConfig
+    const filteredMLSConfig = Object.fromEntries(
+      Object.entries(mlsConfig).filter(([_, value]) => value !== undefined),
+    ) as typeof mlsConfig;
+
     this._config = {
-      ...mlsConfig,
       ...defaultConfig,
+      ...filteredMLSConfig,
     };
```

Comment:
Question: So the main fix is changing the order here, and the secondary fix is to filter out the `undefined` correct?